.golangci.yml

```diff
@@ -0,0 +1,13 @@
+linters:
+  enable:
+    - dogsled
```

Comment:
suggest to add ercheck and govet to the list
https://golangci-lint.run/usage/linters/

Reply:
Those two are active by default, see https://golangci-lint.run/usage/linters/#enabled-by-default-linters. I haven't deactivated them